files/en-us/web/api/mediametadata/index.html

```diff
@@ -52,7 +52,8 @@ <h2 id="Examples">Examples</h2>
       <span class="punctuation token">{</span> src<span class="punctuation token">:</span> <span class="string token">'https://dummyimage.com/384x384'</span><span class="punctuation token">,</span> sizes<span class="punctuation token">:</span> <span class="string token">'384x384'</span><span class="punctuation token">,</span> type<span class="punctuation token">:</span> <span class="string token">'image/png'</span> <span class="punctuation token">}</span><span class="punctuation token">,</span>
       <span class="punctuation token">{</span> src<span class="punctuation token">:</span> <span class="string token">'https://dummyimage.com/512x512'</span><span class="punctuation token">,</span> sizes<span class="punctuation token">:</span> <span class="string token">'512x512'</span><span class="punctuation token">,</span> type<span class="punctuation token">:</span> <span class="string token">'image/png'</span> <span class="punctuation token">}</span><span class="punctuation token">,</span>
     <span class="punctuation token">]</span>
-  <span class="punctuation token">}</span><span class="punctuation token">)</span><span class="punctuation token">;</span></code></pre>
+  <span class="punctuation token">}</span><span class="punctuation token">)</span><span class="punctuation token">;</span>
+  <span class="punctuation token">}</span></code></pre>
```

Comment:
```suggestion
<span class="punctuation token">}</span></code></pre>
```
Needs un-indenting by two spaces